### ISSUE TITLE
Poor Support

### ISSUE THREAD

**Issue by claytonsilva** [meetfranz/franz#2144]
Hi,

I recently purchased an annual license for your product, which would be extremely helpful for my work.

But in the last few months I've been through quite a bit, because of the problem with the windows losing focus.

What makes me very upset about your product is the following:

* You don't answer any issue about the problem saying if you're going to solve it, or at least that you're looking, it's practically abandoned.

* Your site doesn't have a support channel that allows us to contact you, it's a faq and without any email to contact you.

I think you should reconsider and rethink the way you interact with your audience, because so far, I'm using your product because I can't get the money back, after all, how am I going to talk to you if I don't have a clear way to find them

**Comment by claytonsilva** [comment]
https://github.com/meetfranz/franz/issues/2128
https://github.com/meetfranz/franz/issues/2133
https://github.com/meetfranz/franz/issues/2040

**Comment by drodgers-ls** [comment]
I'm just assuming at this point the project has been abandoned.  I started using the native Slack app because of issue 2128, so now Franz is just a shell for FB Messenger.  Not a single dev has appeared in that issue thread or tagged the issue.  Pretty clear that they don't care.

**Comment by jvleminc** [comment]
> I'm just assuming at this point the project has been abandoned. I started using the native Slack app because of issue 2128, so now Franz is just a shell for FB Messenger. Not a single dev has appeared in that issue thread or tagged the issue. Pretty clear that they don't care.

Yeah, I have just switched to Ferdi. https://getferdi.com/ So far all works well.
